api/v1alpha1/nutanix_clusterconfig_types.go

```diff
@@ -49,11 +49,11 @@ type NutanixPrismCentralEndpointSpec struct {
 	// +optional
 	Insecure bool `json:"insecure"`
 
-	// A reference to the ConfigMap containing a PEM encoded x509 cert for the RootCA that was used to create
+	// A base64 PEM encoded x509 cert for the RootCA that was used to create
 	// the certificate for a Prism Central that uses certificates that were issued by a non-publicly trusted RootCA.
 	// The trust bundle is added to the cert pool used to authenticate the TLS connection to the Prism Central.
 	// +optional
-	AdditionalTrustBundle *corev1.LocalObjectReference `json:"additionalTrustBundle,omitempty"`
+	AdditionalTrustBundle *string `json:"additionalTrustBundle,omitempty"`
```

Comment:
jfyi: https://github.com/nutanix-cloud-native/prism-go-client/blob/main/environment/providers/kubernetes/kubernetes.go#L30 we do support string based input as well